python/composio/client/__init__.py

```diff
@@ -335,20 +335,15 @@ def get_connection(
             )
 
         latest_account = None
-        latest_creation_date = datetime.fromtimestamp(0.0)
         connected_accounts = self.client.connected_accounts.get(
             entity_ids=[self.id],
             active=True,
         )
         app = str(app).lower()
         for connected_account in connected_accounts:
             if app == connected_account.appUniqueId:
-                creation_date = datetime.fromisoformat(
-                    connected_account.createdAt.replace("Z", "+00:00")
-                )
-                if latest_account is None or creation_date < latest_creation_date:
-                    latest_creation_date = creation_date
-                    latest_account = connected_account
+                latest_account = connected_account
+                break
```

Comment:
You need to sort the list of connected account by the creation date in reverse fir this logic to work